modules/ROOT/pages/database-administration/standard-databases/listing-databases.adoc

```diff
@@ -82,7 +82,7 @@ The default for a standalone database is `neo4j://localhost:7687`. label:default
 | role
 | The current role of the database (`primary`, `secondary`, `unknown`). label:default-output[]
 
-The value for composite databases is `NULL` because it does not apply to them.
+The value for composite databases is `NULL` because it does not apply to them.footnote:compositeDb[This change applies to versions 2025.04 and later and 5.25.5 and later.]
```

Comment:
also the role column have been null since before, we only updated 3 of the columns but I added the comments in docs on all the columns where it applied as the old ones didn't have docs comments either.
```suggestion
The value for composite databases is `NULL` because it does not apply to them.
```

Reply:
probably, or they were like that from the beginning without being documented